src/record.rs

```diff
@@ -2,6 +2,10 @@ use crate::{data, InitParams, PERFORMANCE_DATA};
 use anyhow::Result;
 use clap::Args;
 use log::{debug, error, info};
+use std::fs;
+use std::os::unix::fs::PermissionsExt;
+
+pub static APERF_TMP: &str = "/tmp/aperf_tmp";
```

Comment:
This should be passed in as a parameter.

Reply:
Do you mean like another CLI option and part of collector params?

Reply:
As part of the collector params passed into each of the data type.